modin/pandas/test/test_io.py

```diff
@@ -60,13 +62,49 @@
 TEST_GBQ_FILENAME = "test_gbq."
 SMALL_ROW_SIZE = 2000
 
+test_data_dir = os.path.join(os.path.dirname(__file__), "read_csv_data")
+if not os.path.exists(test_data_dir):
+    os.mkdir(test_data_dir)
+
+# raising of this exceptions can be caused by unexpected behavior
+# of test, but can passed by eval_io function since the type
+# of this exceptions are the same
+io_ops_bad_exc = [TypeError, FileNotFoundError]
+
+
+def eval_io(
+    fn_name,
+    comparator=df_equals,
+    cast_to_str=False,
+    check_exception_type=True,
+    nonacceptable_exception_types=io_ops_bad_exc,
+    *args,
+    **kwargs,
+):
```

Comment:
Move the utility functions to the appropriate file.

Reply:
Done.